Patches/Caffe/527f97c0692f116ada7cb97eed8172ef7da05416/cmake/Cuda.cmake

```diff
@@ -4,7 +4,7 @@ endif()
 
 # Known NVIDIA GPU achitectures Caffe can be compiled for.
 # This list will be used for CUDA_ARCH_NAME = All option
-if (CUDA_VERSION VERSION_LESS 9.0)
+if (CUDA_VERSION VERSION_LESS 8.0)
```

Comment:
Is this really true. I thought 20 21(20) weren't deprecated until version 9.

Reply:
On the other hand, I guess there's no real harm in removing support for such old GeForce cards, et. al.